### ISSUE TITLE
Add stable option for option -H:Log

### ISSUE THREAD

**Issue by adinn** [oracle/graal#7357]
When command line option `-Log=...` is passed on the native image command line this results in a warning that Log is an experimental option.

```
Warning: The option '-H:Log=DebugTypeInfo:3,DebugCodeInfo:3,DebugDataInfo:3,dwarf:3' is experimental and must be enabled via '-H:+UnlockExperimentalVMOptions' in the future.
```

Experimental? Really?

n.b. although this option results in a specific warning  up front the option is not listed in the later summary of unlocked experimental options that have been unlocked.

```
 3 experimental option(s) unlocked:
 - '-H:-DeleteLocalSymbols' (origin(s): command line)
 - '-H:-OmitInlinedMethodDebugLineInfo' (origin(s): command line)
 - '-H:+SourceLevelDebug' (origin(s): command line)
```

**Comment by christianwimmer** [comment]
Logging the image generator is certainly not a supported feature where we guarantee, e.g., certain logging to always happen or to use a certain specified format. So yes, logging is an option that can change at any time and therefore should not be used in production, i.e., it is experimental.

There might be some confusion about what "experimental" means. The best way is to see it from the opposite direction. Everything that is stable is something that customers can a file a bug about if it is not working in production, and something where we must ensure compatibility in future versions of GraalVM. It has nothing to do with how useful an option is during development or debugging, and it does not imply in way that the option will go away soon (the mechanism to communicate that would be to deprecate an option). An option being experimental simply means "if you choose to use this option in production, you cannot wake me up Sunday night and expect me to fix a problem that blocks your deployment".

**Comment by adinn** [comment]
Ok, understood.

**Comment by fniephaus** [comment]
I'm going to close this for now. https://github.com/oracle/graal/issues/7370 tracks the issue that `-H:Log` is not listed.

**Comment by fniephaus** [comment]
This will be addressed by #7720.
